RoddyCore/src/de/dkfz/roddy/core/CohortDataRuntimeServiceExtension.groovy

```diff
@@ -21,6 +24,10 @@ class CohortDataRuntimeServiceExtension {
 
     private static LoggerWrapper logger = LoggerWrapper.getLogger(CohortDataRuntimeServiceExtension.class.getSimpleName())
 
+    static final String PID_IDENTIFIER = '[\\w*?_-]+'
+    static final String COHORT_IDENTIFIER = "[\\w_-]+"
+    static final String SCOHORT_IDENTIFIER = "([:]${COHORT_IDENTIFIER}[:]){0,1}"
```

Comment:
Why not `SUPERCOHORT_IDENTIFIER`? It's probably not used often and that would be clearer.